test-case/multiple-pause-resume.sh

```diff
@@ -56,7 +56,7 @@ rnd_range=$((rnd_max - rnd_min))
 [[ $rnd_range -le 0 ]] && dlogw "Error random range scope [ min:$rnd_min - max:$rnd_max ]" && exit 2
 
 tplg=${OPT_VAL['t']}
-func_pipeline_export "$tplg" "type:any"
+func_pipeline_export "$tplg" "type:any & ~cap_name:Smart Amplifier Playback 2,Echo Reference Capture 3"
```

Comment:
are we going to skip the combinations of pipeline 2 and other pipelines

Reply:
I am wondering whether it will be better to skip the combination of pipeline 2 and pipeline 3 when constructing pipeline_combine_lst

Reply:
Yes, we can only skip the combination of smart_amp pipeline, but I wonder if such a test would make sense.
eg. check the echo reference capture pipeline with other normal playback pipelines, this is not a reasonable use case.